test/passes/optimize-instructions_fuzz-exec.wast

```diff
@@ -4,57 +4,57 @@
  (func $test32 (export "test32")
   (call $logf32
    (f32.add
-    (f32.const -nan:0xffff82)
+    (f32.const -nan:0x7fff82)
```

Comment:
Why must these nans change?

Reply:
The original NaNs here have invalid payloads. The spec for this is [here](https://webassembly.github.io/spec/core/text/values.html#floating-point).

The `fNMag` production `'nan:0x' n:hexnum => nan(n)` has a side condition `(1 <= n < 2^signif(N))` and `signif(32) = 23`, so the largest allowed NaN payload for a 32-bit float is `2^23 - 1 = 0x7FFFFF`.

Reply:
Thanks, I didn't know that.